docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -37,128 +37,121 @@ To find the `volumeHandle`, use one of the following procedures:
+
+    **Important:** If using the CSI Topology feature, the `spec.csi.volumeHandle` contains the system ID. In the example below, the `spec.csi.volumeHandle` would read similar to the following: `SVC:demo-system-id-1:600507640082000B08000000000004FF`.
+    
+        apiVersion: v1
+        kind: PersistentVolume
+        metadata:
+          # annotations:
+            # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+          name: vol1-pv
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          capacity:
+            storage: 1Gi
+          csi:
+            controllerPublishSecretRef:
+              name: demo-secret
+              namespace: default
+            driver: block.csi.ibm.com
+            # volumeAttributes:
+              # pool_name: ibmc-block-gold
+              # storage_type: SVC
+              # volume_name: vol1
+              # array_address: baremetal10-cluster.xiv.ibm.com
+            volumeHandle: SVC:600507640082000B08000000000004FF
+          # persistentVolumeReclaimPolicy: Retain
+          storageClassName: ibmc-block-gold
+          # volumeMode: Filesystem
+
+3. Create a PersistentVolumeClaim (PVC) YAML file.
 
     **Note:**
 
-    -   To include a specific 5 Gi PV, be sure to include the `storageClassName`.
-    -   For more information about creating a PVC yaml file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+    - To include a specific 5 Gi PV, be sure to include the `storageClassName`.
+    - For more information about creating a PVC YAML file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
     
-    ```screen
-    apiVersion: v1
-    kind: PersistentVolumeClaim
-    metadata:
-      # annotations:
-        # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-      name: vol1-pvc
-    spec:
-      accessModes:
-      - ReadWriteOnce
-      resources:
-        requests:
-          storage: 1Gi
-      storageClassName: ibmc-block-gold
-      volumeName: vol1-pv
-    ```
-
-4.  Create a StatefulSet.
-
-    For more information about creating a StatefulSet, see [Creating a StatefulSet](csi_ug_config_create_statefulset.md).
-
-    ```screen
-    kind: StatefulSet
-    apiVersion: apps/v1
-    metadata:
-      name: sanity-statefulset
-    spec:
-      selector:
-        matchLabels:
-          app: sanity-statefulset
-      serviceName: sanity-statefulset
-      replicas: 1
-      template:
+            apiVersion: v1
+            kind: PersistentVolumeClaim
+            metadata:
+              # annotations:
+                # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+              name: vol1-pvc
+            spec:
+              accessModes:
+              - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 1Gi
+              storageClassName: ibmc-block-gold
+              volumeName: vol1-pv
+
+4. Create a StatefulSet.
```

Comment:
@tyichye19 can I remove the yaml information here and just leave the information text?

Reply:
yes, I agree with Ori

Reply:
DONE

Reply:
I meant that I don't think this step is relevant for importing a volume (the volume won't necessarily be consumed directly by a statefulset, or a pod of any kind for that matter, e.g. it can be used in replication just as backup)

Reply:
since I was counting on referring to this procedure for backward-compatible replication support, it would be problematic to include this step, because a secondary volume in replication should not be consumed in parallel to the primary.
when this step was added between 1.0.0 and 1.1.0, there was no replication, so there was no issue (other than info duplication).

could we at least make this step optional?

Reply:
@oriyarde Do you want me to mark it as optional along with a note?
Note:  Do not use this step when using this procedure with replication support. Replication cannot have a secondary volume being used in parallel to the primary volume.

Reply:
I think it would be better to just remove this step altogether. consuming a volume was never really a part of importing it, and other pages (e.g. replication limitation) should not have a problem referring to this procedure

Reply:
@tyichye19  can you confirm that this can be removed all together? thanks

Reply:
yes I think we should include only the import procedure, and all other procedures such as consuming a volume should be removed.

Reply:
removed statefulset creation.
DONE